src/main/java/com/academy/ui/pages/advancedSearchHeader/AdvancedSearchHeaderComponent.java

```diff
@@ -0,0 +1,70 @@
+package com.academy.ui.pages.advancedSearchHeader;
+
+import com.academy.ui.pages.BaseComponent;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class AdvancedSearchHeaderComponent extends BaseComponent {
+    protected WebElement advancedSearchTextHeading;
+    protected WebElement selectionSearchInputField;
+    protected WebElement selectionSearchInputFieldPlaceholder;
+    protected WebElement searchIcon;
+    protected WebElement advancedSearchIcon;
+    protected AdvancedSearchTooltip advancedSearchTooltip;
+    public AdvancedSearchHeaderComponent(WebDriver driver, WebElement rootElement) {
+        super(driver, rootElement);
+    }
+
+    public WebElement getAdvancedSearchTextHeading() {
+        if (advancedSearchTextHeading == null) {
+            advancedSearchTextHeading = rootElement.findElement(By.xpath("//div[contains(@class, \"city-name-box\")]/h2"));
```

Comment:
```suggestion
            advancedSearchTextHeading = rootElement.findElement(By.xpath("//h2[@class='city-name']"));
```
You can use a straight path